lib/generators/spree/backend/install/install_generator.rb

```diff
@@ -16,7 +16,6 @@ def install
           template 'app/javascript/spree-dashboard.js'
           template 'vendor/assets/javascripts/spree/backend/all.js'
           template 'vendor/assets/stylesheets/spree/backend/all.css'
-          run 'yarn add @spree/dashboard'
```

Comment:
@MatthewKennedy we don't need to remove, yarn link takes precedence - https://classic.yarnpkg.com/en/docs/cli/link